openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -188,13 +194,47 @@ class expanded_income(Variable):
         """Broad income measure that includes benefit_value_total"""
     )
 
+    def formula(tax_unit, period, parameters):
```

Comment:
Expanded income isn't used for tax calculations, but it is what taxcalc uses as their income measure for binning into deciles. Should we include it?

Reply:
I can see benefit in including it (even if it's redundant/unused) for validation later on